src/systems/userinput/resolve-action-sets.js

```diff
@@ -97,10 +99,14 @@ export function resolveActionSets() {
       hcAFRAME(world, "super-spawner", rightRemoteHovering)
   );
 
-  userinput.toggleSet(sets.leftHandHoveringOnVideo, hcAFRAME(world, "media-video", leftHandHovering));
-  userinput.toggleSet(sets.rightHandHoveringOnVideo, hcAFRAME(world, "media-video", rightHandHovering));
-  userinput.toggleSet(sets.leftCursorHoveringOnVideo, hcAFRAME(world, "media-video", leftRemoteHovering));
-  userinput.toggleSet(sets.rightCursorHoveringOnVideo, hcAFRAME(world, "media-video", rightRemoteHovering));
+  userinput.toggleSet(
```

Comment:
Both seemed to be `undefined` in `sets`, so this was calling `toggleSet(undefined, true | false)`. It's a little surprising this didn't cause errors downstream but I'm guessing the `undefined` slipped by unnoticed elsewhere. 

Since we weren't using it, I removed it. Happy to reintroduce if/when we need it.

Reply:
Ah ok that seems fine, just looked like a copy/paste error.